Source/Blazorise/Styles/_breakpoints.scss

```diff
@@ -0,0 +1,29 @@
+﻿body:before {
```

Comment:
I cannot seem to find where body:before is used for Bar. Is there a way for this to be custom class instead, like ".b-body-bar" or something?

Reply:
This code is actually used for the new `BreakpointAdapter` I made.
This adapter allows the component to run code (like the `Bar` needs to) on breakpoint changing.

You can see in `blazorise.js` there is a method (`getBreakpoint`) that calls `window.getComputedStyle(document.body, ':before')` to figure out what the current breakpoint is (this is automatically called by the `window.reszie` event that is configured).

The reason it is on the body tag is because this `BreakpointAdapter` can be used to by any component that needs to run code on breakpoint change (and putting it on `body` doesn't rely on anything else), and I thought it might be useful for the future! 

These are some examples of what I modeled the pattern after: [here](https://www.lullabot.com/articles/importing-css-breakpoints-into-javascript), and [here.](https://gomakethings.com/the-easy-way-to-manage-css-breakpoints-in-javascript/)

Reply:
Also - I was suggestion in my PR that these are generated in the `ThemeGenerator` so they can be easily configured to match a new provider or by Blazorise users

Reply:
I've been thinking of going to do theming. Starting with dark and light theme for Bar and the going with colors.

One thing I'm not sure how to handle. How to override bootstrap, bulma or antdesign breakpoints? We cannot just override them as there are just too many classes that are using them.

Reply:
In regards to the theming, in `_bar.scss` you will see a comment called **Theming** which has `.bar-bar-dark` and `.b-bar-light` under it.. I kept everything relating to Theming wrapped up in those 2.
My plan was to replace everything inside those with CSS generated from the `ThemeGenerator` (as well as the `:root` stuff at the top for sizing).

Yeah, I'm not sure if it will be possible to override them. But with the moving the `_breakpoint.scss` into the `ThemeGenerator`, at least it means it will be easy for us to match them per provider (and perhaps just not make them overrideable by Blazorise users since it will cause too many issues)